skimage/filters/thresholding.py

```diff
@@ -581,8 +581,8 @@ def threshold_li(image):
     tolerance = np.min(np.diff(np.unique(image))) / 2
 
     # Initial estimate
-    t_curr = np.mean(image)
```

Comment:
Can we add a comment here

Reply:
@hmaarrfk ask and ye shall receive. =) Thanks for the quick review!